libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -71,7 +71,7 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @Values{Plane}: 0:Report Only,1:RTL,6:Guided,7:GuidedThrottlePass,8:AUTOLAND if possible else RTL
     // @Values: 0:Report Only,1:RTL or Land
     // @User: Standard
-    AP_GROUPINFO("ACTION",      2,  AC_Fence,   _action,        AC_FENCE_ACTION_RTL_AND_LAND),
+    AP_GROUPINFO("ACTION",      2,  AC_Fence,   _action,        float(Action::RTL_AND_LAND)),
```

Comment:
Are you sure that should be float?

Reply:
Sadly, yes.
![image](https://github.com/user-attachments/assets/251184e5-40f3-4323-b286-b18eaf700ee9)